inline-translation/lib/index.js

```diff
@@ -235,6 +235,10 @@ registerPopoverHandlers = function() {
 					if ( !! document.cookie.match( /inlinejumptonext=1/ ) ) {
 						jQuery( '.translator-translatable.translator-untranslated:visible' ).webuiPopover( 'show' );
 					}
+				} ).fail( function( xhr ) {
+					if ( xhr.responseJSON ) {
+						$form.find( '.warnings' ).html( '<p class="local-inline-warning"><b>Error: </b>' + xhr.responseJSON.message + '</p>' );
```

Comment:
We should avoid using the message directly as HTML (you can omit closing tags here, they will be automatically added).
```suggestion
						$form.find( '.warnings' ).html( '<p class="local-inline-warning"><b>Error: </b><span class="message">');
						$form.find( '.warnings .message' ).text(xhr.responseJSON.message );

```